docs/05-engineering/security-compliance.md

```diff
@@ -48,24 +35,26 @@ We enhance our [Security Awareness with Tools](../09-security/awareness.md) by:
 * We ensure that access to documents/sites/dashboards is limited to those that should have access.
 * This includes our Google Docs!
 * We ensure that users with enhanced privileges (to sites and/or servers)
-* must use TFA for authentication/authorization
-* are appropriately adjusted upon separation from CivicActions.
+  * must use TFA for authentication/authorization
+  * are appropriately adjusted upon separation from CivicActions.
 
 ## Continuous Monitoring
 
 * We use Nagios, Cloudwatch, StatusCake and OpsGenie
 * We implement log analysis procedures that enable better/more timely capture of system anomalies.
```

Comment:
Does this need a "such as:" at the end?  We've used others in certain situations, like New Relic and Pingdom.